demo/index.html

```diff
@@ -32,13 +33,17 @@
   .wide {
     width: 474px;
   }
+  paper-spinner {
+    width: 16px;
+    height: 16px;
+  }
 </style>
 <body>
   <div class="horizontal center-center layout">
     <div>
       <h4>method="get"</h4>
       <div class="horizontal-section">
-        <form is="iron-form" id="formGet" method="get" action="/">
+        <form is="iron-form" id="formGet" method="get" action="https://httpbin.org/delay/2">
```

Comment:
Oh, I was going to merge it if you just made a separate `div`. I'll pick it up and send a separate PR out then, I guess. :)